src/test/a11y/a11y.ts

```diff
@@ -121,5 +121,6 @@ describe('Accessibility', () => {
   checkPaths(OfferPaths)
   checkPaths(StatementOfMeansPaths)
   checkPaths(FullAdmissionPaths)
-  checkPaths(ClaimantResponsePaths)
+  // skipped for demo
+  // checkPaths(ClaimantResponsePaths)
```

Comment:
Should this be commented out any more? the demo is over

Reply:
i have added it back